velox/core/tests/CMakeLists.txt

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_executable(velox_core_test TestMetafunctions.cpp TestString.cpp
-                               TestTypeAnalysis.cpp)
+add_executable(velox_core_test TestMetafunctions.cpp TestQueryConfig.cpp
```

Comment:
> TestMetafunctions.cpp

I wonder what this is for. I looked and it is testing facebook::velox::util::forEachWithIndex function which itself doesn't seem to be used anywhere. It would be nice to delete it along with the test.

Reply:
Sure. Thanks!